ek9000App/src/devEK9000.cpp

```diff
@@ -888,7 +888,8 @@ void ek9000Configure(const iocshArgBuf* args) {
 	devEK9000* dev;
 
 	char ipbuf[64];
-	sprintf(ipbuf, "%s:%i", ip, port);
+	ipbuf[63] = 0;
+	snprintf(ipbuf, sizeof(ipbuf)-1, "%s:%i", ip, port);
```

Comment:
I think [`snprintf`](https://cplusplus.com/reference/cstdio/snprintf/) buffer size accounts for the null terminator, but this is fine regardless

Reply:
You're right- I was thinking of strncpy in my mind when I wrote this